### PR TITLE
Report view shell

### PR DESCRIPTION
**Pull request by Omri-Levy**
- feat(*): added route and link to a single report and added report id
- feat(backoffice-v2): added report view shell

### CODE REVIEW

File: apps/backoffice-v2/src/main.tsx [hunk 1]
```diff
@@ -11,6 +11,10 @@ import { Router } from './Router/Router';
 import { env } from './common/env/env';
 import './i18n';
 import './index.css';
+import dayjs from 'dayjs';
+import advancedFormat from 'dayjs/plugin/advancedFormat';
+
+dayjs.extend(advancedFormat);
```
Comment (liorzam) [inline apps/backoffice-v2/src/main.tsx:17]: what would happen for Chinese

File: apps/backoffice-v2/src/pages/MerchantMonitoring/components/MerchantMonitoringTable/columns.tsx [hunk 2]
```diff
@@ -103,7 +103,7 @@ export const columns = [
                 severityToClassName[
                   (severity?.toUpperCase() as keyof typeof severityToClassName) ?? 'DEFAULT'
                 ],
-                'w-20 py-0.5 font-bold',
```
Comment (liorzam) [inline apps/backoffice-v2/src/pages/MerchantMonitoring/components/MerchantMonitoringTable/columns.tsx]: check

File: apps/backoffice-v2/src/pages/MerchantMonitoring/components/MerchantMonitoringTable/hooks/useMerchantMonitoringTableLogic/useMerchantMonitoringTableLogic.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  const { pathname, search } = useLocation();
+  const locale = useLocale();
+  const onClick = useCallback(() => {
+    sessionStorage.setItem(
+      'merchant-monitoring:business-report:previous-path',
+      `${pathname}${search}`,
+    );
+  }, [pathname, search]);
+  const Cell: IDataTableProps<TBusinessReports>['CellContentWrapper'] = ({ cell, children }) => {
+    const itemId = cell.id.replace(`_${cell.column.id}`, '');
```
Comment (liorzam) [inline apps/backoffice-v2/src/pages/MerchantMonitoring/components/MerchantMonitoringTable/hooks/useMerchantMonitoringTableLogic/useMerchantMonitoringTableLogic.tsx]: take id from row

File: apps/backoffice-v2/src/pages/MerchantMonitoringBusinessReport/MerchantMonitoringBusinessReport.page.tsx [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+  });
+  const [{ activeTab }] = useZodSearchParams(MerchantMonitoringBusinessReportSearchSchema);
+  const navigate = useNavigate();
+  const onNavigateBack = useCallback(() => {
+    const previousPath = sessionStorage.getItem(
+      'merchant-monitoring:business-report:previous-path',
+    );
+
+    if (!previousPath) {
+      navigate('../');
```
Comment (liorzam) [inline apps/backoffice-v2/src/pages/MerchantMonitoringBusinessReport/MerchantMonitoringBusinessReport.page.tsx:70]: validate when a user get into this page by link.

File: apps/backoffice-v2/src/main.tsx (hunk 1 above)
Comment (Omri-Levy) [reply apps/backoffice-v2/src/main.tsx:17]: `dayjs.format` controls the format for all locales.

**Review by liorzam — APPROVED** [review]

File: apps/backoffice-v2/src/pages/MerchantMonitoringBusinessReport/MerchantMonitoringBusinessReport.page.tsx (hunk 4 above)
Comment (Omri-Levy) [reply apps/backoffice-v2/src/pages/MerchantMonitoringBusinessReport/MerchantMonitoringBusinessReport.page.tsx:70]: This should be handled only for drawers, dialogs, modals, sheets, and so on. In this case its a go back button.

**Pull request merged**